into/convert.py

```diff
@@ -79,6 +79,22 @@ def numpy_chunks_to_numpy(c, **kwargs):
     return np.concatenate(list(c))
 
 
+@convert.register(chunks(np.ndarray), np.ndarray, cost=0.5)
+def numpy_to_chunks_numpy(x, chunksize=2**20, **kwargs):
+    return chunks(np.ndarray)(
+            lambda: (x[i:i+chunksize] for i in range(0, x.shape[0], chunksize)))
+
+
+@convert.register(pd.DataFrame, chunks(pd.DataFrame), cost=1.0)
+def chunks_dataframe_to_dataframe(c, **kwargs):
+    return pd.concat(list(c), axis=0)
```

Comment:
i would pass `**kwargs` in here and maybe put an `axis=0` default in the signature. there are other kwargs that are useful here: 

http://pandas.pydata.org/pandas-docs/stable/generated/pandas.tools.merge.concat.html